lesson3/Lesson_3/app/src/main/res/layout/activity_main.xml

```diff
@@ -1,12 +1,12 @@
 <?xml version="1.0" encoding="utf-8"?>
-<android.support.design.widget.CoordinatorLayout
-    android:id = "@+id/container"
+<LinearLayout
```

Comment:
Your toolbar was overlapping with your fragment, which made it so I couldn't see your "To Do List" TextView at the top of your MainActivityFragment!  Make sure when you're testing your code that you visually test it to make sure it's displaying what you actually want.  This is a fix for your overlapping problem, use LinearLayout with a vertical orientation instead of a Coordinator Layout.